jsonnet/components/alertmanager.libsonnet

```diff
@@ -76,12 +76,13 @@ function(params)
         } + withDescription(
           |||
             Expose the Alertmanager web server within the cluster on the following ports:
-            * Port %d provides access to all the Alertmanager endpoints. %s
+            * Port %d provides access to all the Alertmanager endpoints. %s %s
```

Comment:
can you make it look like for uwm alertmanager?
https://github.com/openshift/cluster-monitoring-operator/blob/6e85d9ddf1296b0013d10c1a2453f2dd639520e0/jsonnet/components/alertmanager-user-workload.libsonnet#L40-L47